imports/personal-access-tokens/server/publications/personalAccessTokens.js

```diff
@@ -3,6 +3,7 @@ import { Meteor } from 'meteor/meteor';
 import { hasPermission } from '../../../../app/authorization';
 import { Users } from '../../../../app/models';
 
+console.warn('The publication "personalAccessTokens" is deprecated and will be removed after version v3.0.0');
```

Comment:
Can we move those warnings to inside the publication, so then we know when it has being used, today ir raises all those warnings on server startup what is not meaningful to understand if someplace is still using it.

Reply:
@rodrigok That was a suggestion from @ggazzo and @sampaiodiego.

Reply:
right.. this way (outside the publication) there is more chance of and admin to see the warning.

I would not mind adding to both places though. wdyt?